jenkins/job-configs/kubernetes-jenkins/kubernetes-anywhere-e2e.yaml

```diff
@@ -0,0 +1,62 @@
+- job-template:
+    name: 'kubernetes-anywhere-e2e-{suffix}'
```

Comment:
Can you make the prefix mikedanese or something so that its test nature is more obvious?